third_party/move/move-model/src/model.rs

```diff
@@ -821,10 +821,13 @@ impl GlobalEnv {
         target_modules
     }
 
-    fn add_backtrace(msg: &str, is_bug: bool) -> String {
+    fn add_backtrace(msg: &str, _is_bug: bool) -> String {
+        // For now, we do not use is_bug, but we could have
+        // another env var MOVE_COMPILER_DEBUG_BUG_ENV_VAR to
```

Comment:
nit: we should start using `MVC_..` instead of `MOVE_COMPILER_..` for all related env vars.

Given that it is not a new problem that this PR introduces, we could also do this later. Flagging here just for discussion.

Reply:
Agreed, but this is for another time. I don't use that var BTW its @brmataptos who introduced this.